Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -1487,6 +1489,28 @@ QString Scene_polyhedron_item::computeStats(int type)
     else
       return QString("n/a");
   }
+  case GENUS:
+  {
+    if(!d->poly->is_closed())
+    {
+      return QString("n/a");
+    }
+    else if(d->genus == -std::numeric_limits<double>::infinity())
+    {
+      std::size_t S(d->poly->size_of_vertices()), A(d->poly->size_of_halfedges()/2), F(d->poly->size_of_facets());
+      //cast in int to avoid intermediary state containing a negative size_t.
+      d->genus = 1.0 - (static_cast<int>(S)-static_cast<int>(A)+static_cast<int>(F))/2.0;
```

Comment:
Let's say the type of `d->genus` is `std::ptrdiff_t`, then:
```c++
std::ptrdiff_t S(d->poly->size_of_vertices()), A(d->poly->size_of_halfedges()/2), F(d->poly->size_of_facets());
d->genus =  1 - (S-A+F)/2;
```

I'd like also to remind that the naming convention of CGAL says that variable names should be lowercase.